parser.go

```diff
@@ -20,6 +20,8 @@ var ErrRead = errors.New("short read - malformed packet?")
 
 // A Message is the top level construct representing an IPFIX message. A well
 // formed message contains one or more sets of data or template information.
+// The array "DataRecords" stores the actual IPFIX data while "TemplateRecords"
+// stores the corresponding template at the same index.
```

Comment:
I don't think this is correct. There's no relation between the template ID and the index in the TemplateRecords slice.